### PR TITLE
feat(discoverv2): removing issue from tab and updating order of issue in sidebar

### PR DESCRIPTION
**Pull request by doralchan**
Screen Shot 2019-11-26 at 11 17 02 AM

### CODE REVIEW

**Review by markstory — APPROVED** [review]

File: src/sentry/static/sentry/app/views/eventsV2/eventDetails/content.tsx [hunk 1]
```diff
@@ -262,8 +262,8 @@ const EventHeader = (props: {event: Event}) => {
   const {title} = getTitle(props.event);
   return (
     <StyledEventHeader data-test-id="event-header">
-      <StyledTitle>{title}</StyledTitle>
-      <span>{getMessage(props.event)}</span>
+      <StyledTitle>{title}:</StyledTitle>
```
Comment (dashed) [inline src/sentry/static/sentry/app/views/eventsV2/eventDetails/content.tsx]: @doralchan `<StyledTitle />` should also be `gray2` as per Cramer's feedback.

**Review by dashed — CHANGES_REQUESTED** [review]

**Comment by doralchan** [comment]
I did change that! 

Screen Shot 2019-11-26 at 3 07 45 PM

**Review by dashed — APPROVED** [review]
Looks good to me 👍

File: src/sentry/static/sentry/app/views/eventsV2/eventDetails/content.tsx (hunk 1 above)
Comment (dashed) [reply src/sentry/static/sentry/app/views/eventsV2/eventDetails/content.tsx]: Oh oops. looks like it was addressed in this PR now.

**Pull request merged**